recipes/r-getpass/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = '0.2-2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-getpass
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: getPass_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/getPass_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/getPass/getPass_{{ version }}.tar.gz
+  sha256: 8cfdc7e8cabeae5dd325015d735b139d9828bd54ea8e7c2dcb636a00a153cd0f
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-rstudioapi >=0.5
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-rstudioapi >=0.5
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('getPass')"  # [not win]
+    - "\"%R%\" -e \"library('getPass')\""  # [win]
```

Comment:
Isn't `R -e "library('getPass')"` supposed to work across platforms?

Reply:
Mh, this is what the skeleton generator is giving us.

Reply:
@isuruf How important is this to you? Fixing this automatically via the [helper script](https://github.com/bgruening/conda_r_skeleton_helper) we use would take some effort because it would require obtaining the correct capitalization of the R package. I'm not saying it cannot be done, but I don't think it's worth the effort.

Reply:
It was just an observation. Not important at all